packages/@office-iss/react-native-win32-tester/overrides.json

```diff
@@ -26,6 +26,10 @@
       "baseFile": "packages/rn-tester/js/components/RNTesterExampleFilter.js",
       "baseHash": "e0c08f622a9d7d12e2115ec16698c8612572a8e1"
     },
+    {
+      "type": "platform",
```

Comment:
Could you choose "patch" instead of "platform" so that upstream changes are merged into this?